frame/wormhole/backing/s2s/src/lib.rs

```diff
@@ -154,11 +175,13 @@ pub mod pallet {
 	pub type SecureLimitedRingAmount<T> =
 		StorageValue<_, (RingBalance<T>, RingBalance<T>), ValueQuery>;
 
+	/// `(asset_address, sender, amount)` the user *sender* lock and remote issuing amount of asset
 	#[pallet::storage]
-	#[pallet::getter(fn locked_queue)]
-	pub type LockedQueue<T: Config> =
-		StorageMap<_, Identity, TokenMessageId, (AccountId<T>, Token), ValueQuery>;
+	#[pallet::getter(fn transaction_infos)]
+	pub type TransactionInfos<T: Config> =
+		StorageMap<_, Identity, TokenMessageId, (H160, AccountId<T>, RingBalance<T>), ValueQuery>;
```

Comment:
1. This storage tuple has three items, considering give it a struct?
2. Please check if this change needs a migration since the pallet already running in the test net.

Reply:
I want to make a migration before. But here this storage is a temporary store. And another reason, the old type of this storage has been deleted. If we make this migration, we must keep the old type in this version, and delete it the next version, it's more complex.
And use the tuple because I don't want to add another type about this storage.

Reply:
A simple way is to add a new storage item named `TransactionInfos` using a new struct and delete the staled old storage item `LockedQueue` using data migration. Delete data using migration is easy to work. I think the most important thing is how we define this storage item better in the production env. We must have a way(even do nothing) to deal with this data migration in the testnet.

Reply:
I have removed the first unused item.